packages/utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@connext/nxtp-utils",
-  "version": "2.5.0-alpha.1",
```

Comment:
are we skipping a version here?

Reply:
Yes.
Syncing with sub version numbers to be the same with the other packages